compiler/src/varalloc.ml

```diff
@@ -262,26 +262,26 @@ let init_slots pd stack_pointers alias coloring fv =
   !slots, lalloc
 
 (* --------------------------------------------------- *)
-let all_alignment pd (ctbl: alignment) alias ret params lalloc : param_info option list * wsize Hv.t =
+let all_alignment pd (ctbl: alignment) alias params lalloc : param_info option list * wsize Hv.t =
 
   let get_align c =
     try Hv.find ctbl c.Alias.in_var
     with Not_found -> no_alignment_constraint in
-  let doparam i x =
+  let doparam x =
     match x.v_kind with
     | Reg (_, Direct) -> None
-    | Reg (_, Pointer _) ->
+    | Reg (_, Pointer writable) ->
       let c = Alias.normalize_var alias x in
       assert (V.equal x c.in_var && c.scope = E.Slocal);
       let pi_ptr = 
         match Hv.find lalloc x with
         | RegPtr p -> p
         | _ | exception Not_found -> assert false in
-      let pi_writable = List.mem (Some i) ret in
+      let pi_writable = writable = Writable in
```

Comment:
May you chose a different variable name? The strings `writable` and `Writable` look so similar.

Reply:
Done.